docs/source/operations/howto/frontend-deployment.rst

```diff
@@ -0,0 +1,51 @@
+**********************************
+How to deploy the Substra frontend
+**********************************
+
+This page contains info about deploying the frontend component of Substra.
+
+Prerequisites
+=============
+
+The Substra frontend is a standalone Helm chart that only needs to be told under what URL the backend API is to be contacted.
+
+This means you first need :doc:`a running backend deployment <backend-deployment>`, which must accept connections on an accessible URL.
+
+You'll need to tell the backend to set the proper headers for cross-origin resources, in the **backend values**:
+
+.. code-block:: yaml
+
+   config:
+     CORS_ORIGIN_WHITELIST: '["your.frontend.url"]' # this is a string parsed as a JSON list
+     CORS_ALLOW_CREDENTIALS: True
+
+Preparing your Helm values
+==========================
+
+You'll need to specify the backend API url:
```

Comment:
If I understand correctly (by comparison with the documentation for deploying the backend), one needs to add these lines (and the ones concerning the ingress) in a file called `frontend-values.yml'  for example?

Reply:
Yes, exactly. I figured people would have done the backend part first so it didn't bear repeating.